src/concrete/ml/sklearn/base.py

```diff
@@ -1319,7 +1345,7 @@ def fit(self, X: Data, y: Target, **fit_parameters):
         # Check that the underlying sklearn model has been set and fit
         assert self.sklearn_model is not None, self._sklearn_model_is_not_fitted_error_message()
 
-        # Convert the tree inference with Numpy operators
+        # Enable rounding feature
         enable_rounding = os.environ.get("TREES_USE_ROUNDING", "1") == "1"
```

Comment:
Still env var based

Reply:
this one, is for rounding, not fhe_sum.

If we decide to remove it as well, it should be in another PR.